src/languages/en.ts

```diff
@@ -1856,6 +1856,7 @@ export default {
             addRate: 'Add rate',
             workspaceDefault: 'Workspace currency default',
             foreignDefault: 'Foreign currency default',
+            customTaxName: 'Custom tax name',
```

Comment:
@filip-solecki Did we confirm this translation before?

Reply:
Yep it is confirmed version